util/validator.go

```diff
@@ -0,0 +1,11 @@
+package util
+
```

Comment:
seems useless code here?

Reply:
removed as use `msg.ValidateBasic` for format validation